chemprop/uncertainty/calibrator.py

```diff
@@ -257,12 +287,14 @@ def fit(self, uncs: Tensor, targets: Tensor, mask: Tensor) -> Self:
         Parameters
         ----------
         uncs: Tensor
-            the predicted uncertainties (i.e., the predicted probabilities for each class) of the shape of ``n x t x c``, where ``n`` is the number of input
-            molecules/reactions, ``t`` is the number of tasks, and ``c`` is the number of classes.
+            the predicted uncertainties (i.e., the predicted probabilities for each class) of the
```

Comment:
I also make these lines shorter to stay less than 100 charactes

Reply:
Yeah, I don't think flake8 looks at the length of strings.

Reply:
It seems that it is due to we ignore the E501 error in .flake8 file
https://github.com/chemprop/chemprop/blob/main/.flake8#L2